cli/commands/entity/info.go

```diff
@@ -43,11 +43,8 @@ func ShowCommand(cli *cli.SensuCli) *cobra.Command {
 				if err := helpers.PrintJSON(r, cmd.OutOrStdout()); err != nil {
 					return err
 				}
-			} else {
```

Comment:
The removal of all of these elses from InfoCommands is what's causing this behavior.